pysal/contrib/viz/plot.py

```diff
@@ -0,0 +1,47 @@
+"""
+Moran Plot using PySAL and Matplotlib
+
+"""
+
+__author__ = "Marynia Kolak <marynia.kolak@gmail.com>"
+
+import pandas as pd
+import numpy as np
+import pysal as ps
+import matplotlib.pyplot as plt
+
+
+def Moran(var, w, **kwargs):
+	'''
+    Produce basic Moran Plot 
+    ...
+
+    Arguments
+    ---------
+
+    var	         	: array
+                      values of variable
+    w       	    : array
+                      values of spatial weight
+    '''
+	w.transform = 'r'
+	slag = ps.lag_spatial(w, var)
+
+	 # Z-Score standardisation
+	y_std   = (var - var.mean())/var.std()
+	yl_std  = (slag - slag.mean())/slag.std()
+
+	fig1 = plt.figure(figsize=(5,5))
+
+	plt.scatter(y_std, yl_std, s=60, color='k', alpha=.6)
+
```

Comment:
We might want to add a linear (and potentially non-linear) fit line. Also, maybe we could add a legend for that line that includes the coefficient for Moran's I (as it's its slope)?

At the very least, I'd add the linear fit and maybe introduce an option for a LOESS line, although that'd make us depend on `statsmodels`.